content/docs/hooks-overview.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: एक नज़र में Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+React 16.8 में *Hooks* नामक एक नया फीचर ऐड हुआ है। यह आपको class का इस्तेमाल किये बिना state और अन्य React विशेषताओं का उपयोग करने देता है।
 
-Hooks are [backwards-compatible](/docs/hooks-intro.html#no-breaking-changes). This page provides an overview of Hooks for experienced React users. This is a fast-paced overview. If you get confused, look for a yellow box like this:
+Hooks [बैकवर्डस-कम्पेटिबल](/docs/hooks-intro.html#no-breaking-changes) है। यह पेज अनुभवी React उपयोगकर्ताओं के लिए Hooks का ओवरव्यू प्रदान करता है। यह एक तेज गति का ओवरव्यू है। यदि आप उलझन में पड़ जाते हैं, तो निचे दिखाए गए पीले बॉक्स की तलाश करे:
 
->Detailed Explanation
+>विस्तृत विवरण
 >
->Read the [Motivation](/docs/hooks-intro.html#motivation) to learn why we're introducing Hooks to React.
+>यह जानने के लिए [मोटिवेशन](/docs/hooks-intro.html#motivation) पढ़ें कि हम Hooks का React में परिचय क्यों कर रहे हैं।
```

Comment:
revisit grammar of the sentence

Reply:
could be reworded as:

Hooks ke React main parichay hone ke karanon ko jaanne ke liye motivation padhen